### PR TITLE
Add battery-state binary_sensors, translate battery-low message to a state-change

### PR DESCRIPTION
**Pull request by bouwew**
## Summary by CodeRabbit

- **New Features**
	- Introduced `binary_sensors` to track battery state for multiple devices, enhancing monitoring capabilities.
	- Updated item counts across various configurations to reflect the increased number of devices managed.
	- Enhanced device management functionality with integrated low battery detection.

- **Bug Fixes**
	- Addressed potential inaccuracies in device battery monitoring with newly implemented checks.

- **Tests**
	- Expanded test suite for improved coverage of device interactions, ensuring reliability with the latest changes.

- **Documentation**
	- Added XML notifications for low battery status, enhancing user awareness of device health.

### CODE REVIEW

**Comment by bouwew** [comment]
@CoMPaTech what happened to the blacking of long lines? Is that now gone with Ruff?
Or has the max width increased by a lot?

**Review by CoMPaTech — APPROVED** [review]
Looks okay code wise so far - lets check actions in the morning why some fail

File: plugwise/data.py [hunk 1]
```diff
@@ -52,13 +54,40 @@ def _update_gw_devices(self) -> None:
...
+                mac_address: str | None = None
+                message: str | None = notification.get("message")
+                if message is not None and all(x in message for x in matches):
+                    mac_address = re.findall(mac_pattern, message)[0]  # re.findall() outputs a list
+
+                if mac_address is not None:
+                    self._notifications.pop(msg_id)
+                    mac_address_list.append(mac_address)
+
+        return mac_address_list
```
[the remark this answers is not in the document]
Comment (bouwew) [reply plugwise/data.py]: Implemented as suggested.

File: plugwise/data.py [hunk 2]
```diff
@@ -52,13 +54,45 @@ def _update_gw_devices(self) -> None:
...
+                mac_address: str | None = None
+                message: str | None = notification.get("message")
+                if message is not None and all(x in message for x in matches) and (mac_addresses := re.findall(mac_pattern, message)):
+                    mac_address = mac_addresses[0]  # re.findall() outputs a list
+
+                if mac_address is not None:
+                    self._notifications.pop(msg_id)
+                    mac_address_list.append(mac_address)
+
+        return mac_address_list
```
[the remark this answers is not in the document]
Comment (bouwew) [reply plugwise/data.py]: Implemented.

**Comment by bouwew** [comment]
Let's release to v1.1.0a0 for testing purposes.

**Pull request merged**